easytest/noncompliant

```diff
@@ -0,0 +1,8 @@
+"""
+add some non pep8 compliant code to test stickler-ci
+"""
+
+
+def foo(a):      
+ x=5.
+ b    =    [    'Hello'    +    'World']
```

Comment:
o.k., that worked now. I simply did not recognize that I had forgotten the file extension. Thanks @bjlittle